pimd/pim_mlag.c

```diff
@@ -965,8 +965,12 @@ void pim_mlag_register(void)
 
 	router->mlag_process_register = true;
 
+#if PIM_IPV == 6
+	/* TBD pim6_mlag_register_handler */
```

Comment:
Are we 100% positive code to fill this is going to be forthcoming very soon? I'm concerned these blocks may end up just sitting here for a decade with no usage.

Reply:
As part of pimv6 mlag work, we will develop this code.